instrumentation/opentelemetry-instrumentation-aws-lambda/tests/test_aws_lambda_instrumentation_manual.py

```diff
@@ -379,3 +380,67 @@ def test_api_gateway_http_api_proxy_event_sets_attributes(self):
                 SpanAttributes.HTTP_USER_AGENT: "agent",
             },
         )
+
+    def test_successful_request_hook(self):
+        lambda_event_attribute = "lambda.event"
+        lambda_event = {"abcd": 1234}
+
+        def request_hook(span, event, _context):
+            span.set_attribute(lambda_event_attribute, json.dumps(event))
+
+        AwsLambdaInstrumentor().instrument(request_hook=request_hook)
+
+        mock_execute_lambda(lambda_event)
+
+        spans = self.memory_exporter.get_finished_spans()
+        assert spans
+        self.assertEqual(len(spans), 1)
+        span = spans[0]
+        self.assertSpanHasAttributes(
+            span,
+            {
+                lambda_event_attribute: json.dumps(lambda_event),
+            },
+        )
+
+    def test_response_hook(self):
+        lambda_response_attribute = "lambda.response"
+        lambda_error_attribute = "lambda.has_error"
+
+        def response_hook(span, result, error):
+            span.set_attribute(lambda_response_attribute, result)
+            span.set_attribute(lambda_error_attribute, error is not None)
+
+        AwsLambdaInstrumentor().instrument(response_hook=response_hook)
+        mock_execute_lambda()
+
+        spans = self.memory_exporter.get_finished_spans()
+        assert spans
+        self.assertEqual(len(spans), 1)
+        span = spans[0]
+        self.assertSpanHasAttributes(
+            span,
+            {
+                lambda_response_attribute: "200 ok",
+                lambda_error_attribute: False,
+            },
+        )
+
```

Comment:
Add success test for both request_hook and response_hook

Reply:
Already done, see the two first tests